addons/mail/static/src/core/common/thread.xml

```diff
@@ -24,13 +24,11 @@
                     <div t-if="msg.threadAsFirstUnread?.eq(props.thread)" class="o-mail-Thread-newMessage d-flex align-items-center fw-bolder z-1">
                         <hr class="flex-grow-1 border-danger opacity-100"/><span class="ps-2 pe-1 bg-danger o-text-white rounded text-uppercase">New</span>
                     </div>
-                    <t t-if="msg.isNotification">
-                        <t t-call="mail.NotificationMessage"/>
-                    </t>
-                    <Message t-else=""
+                    <Message
                         className="getMessageClassName(msg)"
                         isInChatWindow="props.isInChatWindow"
                         message="msg"
+                        previousMessage="msg"
```

Comment:
`previousMessage` should be `prevMsg`, not `msg`, I suppose?

Reply:
Good catch, I'm concerned that the test suite passes :sweat: